com/course/zenkouv/src/main/java/lecture6/GitHubApplication.java

```diff
@@ -0,0 +1,25 @@
+package lecture6;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import lecture6.constant.GitHubApiConstants;
+import lecture6.executors.httpComponent.HttpComponentExecutor;
+import lecture6.gitHub.GitHub;
+import lecture6.helper.ParseHelper;
+import lecture6.model.Contributors;
+import lecture6.model.GitHubUser;
+
+public class GitHubApplication {
+
+    public static void main(String[] args) throws IOException {
+        Map<String, String> params = new HashMap<>();
+        params.put("access_token", GitHubApiConstants.ACCESS_TOKEN);
+        GitHub gitHub = new GitHub(new HttpComponentExecutor());
```

Comment:
i suggest to put `ACCESS_TOKEN ` to `GitHub ` constructor.
you shouldn't operate such technical object as Map in main

Reply:
you may create you map with param in controller